### ISSUE TITLE
New Project creation doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#8]
```
What steps will reproduce the problem?
1. OSX 10.6.5, 32 bit, Eclipse Helios Service Release 1, Android Development 
Toolkit 8.0.1, your phonegap plugin v 1.1
2. Follow instructions on 
http://wiki.phonegap.com/w/page/34483744/PhoneGap-Eclipse-PlugIn-for-Android

3. The last step of the instructions is "Hit Finish to complete"

When I press Finish, the dialog does not dismiss. It stays up. A default 
Android project is created, but the project doesn't have any PhoneGap-specific 
data, it's just the default Android project.

I'm guessing your Phonegap-specific hooks are not getting called after the 
Android project is created.


```

Original issue reported on code.google.com by `jack...@google.com` on 14 Jan 2011 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
Would you do About Eclipse -> Installation Details.  Also, if you started 
Eclipse from a shell, does anything show up there?

I'm running the same config except for OSX 10.6.4.  I'll see if upgrading makes 
a difference for me.
```

Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 3:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
  Android DDMS  8.0.1.v201012062107-82219   com.android.ide.eclipse.ddms.feature.group
  Android Development Tools 8.0.1.v201012062107-82219   com.android.ide.eclipse.adt.feature.group
  Android Hierarchy Viewer  8.0.1.v201012062107-82219   com.android.ide.eclipse.hierarchyviewer.feature.group
  AnyEditTools  2.3.4.201012191830  AnyEditTools.feature.group
  Eclipse IDE for Java Developers   1.3.1.20100916-1202 epp.package.java
  JavaScript Development Tools  1.2.3.v201010160635-7C78FGUF9JgLWLQ1UIPrac  org.eclipse.wst.jsdt.feature.feature.group
  Web Page Editor (Optional)    2.3.1.v20100906-4619oB5856D8Q3324   org.eclipse.jst.webpageeditor.feature.feature.group

Plus your 1.1 plugin (I removed it more recently because it didn't work.)
```

Original comment by `jack...@google.com` on 20 Jan 2011 at 6:03

**Comment by GoogleCodeExporter** [comment]
```
I updated to OSX 10.6.6 and reproduced the plugin environment, but still can't 
reproduce the issue.  See below.   

Let me know if you see anything else different or if anyone else has the 
problem.   The only thing I can think of is some kind of threading issue.  

  Android DDMS  8.0.1.v201012062107-82219   com.android.ide.eclipse.ddms.feature.group
  Android Development Tools 8.0.1.v201012062107-82219   com.android.ide.eclipse.adt.feature.group
  Android Hierarchy Viewer  8.0.1.v201012062107-82219   com.android.ide.eclipse.hierarchyviewer.feature.group
  AnyEditTools  2.3.4.201012191830  AnyEditTools.feature.group
  Eclipse IDE for Java Developers   1.3.1.20100916-1202 epp.package.java
  JavaScript Development Tools  1.2.3.v201010160635-7C78FGUF9JgLWLQ1UIPrac  org.eclipse.wst.jsdt.feature.feature.group
  jQuery Mobile License 1.0.0.201101121603  com.mds.jqm.license.feature.group
  PhoneGap License  1.0.0.201101121603  com.mds.phonegap.license.feature.group
  PhonegapForAndroid    1.1.1.201101121603  com.mds.phonegapForAndroid.feature.group
  Web Page Editor (Optional)    2.3.1.v20100906-4619oB5856D8Q3324   org.eclipse.jst.webpageeditor.feature.feature.group
```

Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 4:52

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Well, I don't know if that's the whole problem, because I seem to have Eclipse 
IDE for Java Developers, rather than Eclipse Java EE:

Eclipse IDE for Java Developers

Version: 1.3.1.20100916-1202
Build id: 20100917-0705

(c) Copyright Eclipse contributors and others 2000, 2010.  All rights reserved.
Visit http://www.eclipse.org/platform

This product includes software developed by the
Apache Software Foundation http://www.apache.org/
```

Original comment by `jack...@google.com` on 20 Jan 2011 at 5:35

**Comment by GoogleCodeExporter** [comment]
```
Oops.  I put comment 5 on the wrong issue.  It should be for Issue 7.  Sorry.
```

Original comment by `paul.beu...@gmail.com` on 20 Jan 2011 at 5:46
- Changed title: **New Project creation doesn't work**

**Comment by GoogleCodeExporter** [comment]
```
Doesn't work for me either. Tried phonegap download and phonegap github. 
Uneducated guess: New wizard makes outdated assumptions on the directory 
structure.
```

Original comment by `matthias...@gmail.com` on 1 Feb 2011 at 11:14

**Comment by GoogleCodeExporter** [comment]
```
@matthias - I just checked both phonegap directory structures. There have been 
no recent changes and today's download works fine for me.

Will you provide more details.  Any error messages? What's in your Eclipse 
Installation Details?
```

Original comment by `paul.beu...@gmail.com` on 1 Feb 2011 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
Error message in the wizard: 
If I point it 
- to the official phonegap download: For the "Location: must be a valid 
directory containing an index.html file"
- to the example directory of the github clone: Invalid phonegap-android 
location. If it's from github,it should have a framework and example 
subdirectory.If it's from www.phonegap.com Download, it should have an Android 
subdirectory
```

Original comment by `matthias...@gmail.com` on 1 Feb 2011 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
The error you're seeing with the 0.9.3 is an invalid source directory.  You can 
click to "Use phonegap example source as template for project" and it should 
update the Location: to Android/Sample/assets/www which should contain an 
index.html

$ ls phonegap-0.9.3/Android/Sample/assets/www
index.html      main.js         master.css      phonegap-0.9.3.js

For github, my directory downloaded and unzipped from github today does include 
an example and framework directory:

$ ls phonegap-android110201
LICENSE     README.md   bin     example     framework   lib     util

What do you see in the directory that you're specifying for the 
phonegap-android Location?



```

Original comment by `paul.beu...@gmail.com` on 1 Feb 2011 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
@Jack I may have finally reproduced your problem.  The dialog does not dismiss 
if there is no Project Explorer window open.  Thus a workaround for the issue 
is to do Window->Show View ->Project Explorer before project creation.

I'll fix the problem of the dialog not dismissing with no Project Explorer in 
the 1.2 release due out next week
```

Original comment by `paul.beu...@gmail.com` on 18 Feb 2011 at 6:20
- Added labels: **Milestone-Release1.2.0**

**Comment by GoogleCodeExporter** [comment]
```
Fix checked in 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=66
```

Original comment by `paul.beu...@gmail.com` on 21 Feb 2011 at 2:23
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
